src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/domain/NewCourse.kt

```diff
@@ -3,6 +3,7 @@ package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.domain
 data class NewCourse(
   val name: String,
   val description: String,
+  val identifier: String,
```

Comment:
I think there's a typo in the commit message here - you've used `identity` rather than `identifier` - I know, technically, the word has the same meaning in this case, but might be clearer to use `identifier` if you wouldn't mind rewording it.

Reply:
You're right, I got mixed up in the code as well, then realised what I'd done.  I think the code is all correct (it must be otherwise it wouldn't compile and pass the tests).  Clearly I used the wrong term in the commit message.